src/pgrn-custom-scan.c

```diff
@@ -781,6 +781,8 @@ PGrnMakeTupleTableSlot(CustomScanState *customScanState,
 	ListCell *cell;
 	ExecClearTuple(slot);
 	econtext->ecxt_scantuple = slot;
+	Snapshot snapshot = SnapshotAny;
+	snapshot = RegisterSnapshot(GetTransactionSnapshot());
```

Comment:
```suggestion
	Snapshot snapshot = RegisterSnapshot(GetTransactionSnapshot());
```

Thanks for your contribution!
Could you assign `RegisterSnapshot(GetTransactionSnapshot())` directly to `snapshot`.

Reply:
sure